script/input/1/mainnet-staging.json

```diff
@@ -0,0 +1,14 @@
+{
+    "admin": "0x6F3066538A648b9CFad0679DF0a7e40882A23AA4",
+    "allocatorVault": "0x8E20650287635aE6e20ce38EcD3E795919D52354",
+    "almProxy": "0xC29D06ce81137E6B3C3DC090713636d81600a347",
+    "cctpTokenMessenger": "0xBd3fa81B58Ba92a82136038B25aDec7066af3155",
+    "dai": "0x6B175474E89094C44Da98b954EedeAC495271d0F",
+    "daiUsds": "0x3225737a9Bbb6473CB4a45b7244ACa2BeFdB276A",
+    "psm": "0xf6e72Db5454dd049d0788e411b06CfAF16853042",
```

Comment:
Same here. I thought we use a mock for psm. Need to find this address and replace.

Reply:
Updated with mock PSM